src/pages/host/hostDetail/detail.jsx

```diff
@@ -0,0 +1,78 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Outlet } from "react-router-dom";
+import { HiArrowLeft } from "react-icons/hi";
+import HostDetailNav from "../../../components/host/hostDetailNav";
+
+export default function HostVanDetail() {
+  const [hostList, setHostList] = useState([]);
+  const listId = useParams();
+  const item = hostList[listId.id];
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const detailFetcher = async () => {
+      try {
+        const detail = await fetch("/api/vans");
+        const result = await detail.json();
+        setHostList(result.vans);
+        setLoading(false);
+      } catch (e) {
+        setHostList("Nothing found I ain't gon lie");
+        setLoading(false);
+      }
+    };
+    detailFetcher();
+    return () => {
+      detailFetcher();
+    };
```

Comment:
Is there a reason you are fetching again in the cleanup function?

Reply:
DONE :heavy_check_mark: